magento/pwa-studio
# [Doc] Release note 7.0.0

## Description

Update the release notes for the 7.0.0 release.

## Related Issue





Closes PWA-719

## Acceptance 

- @awilcoxa 
- @cherdman

### Verification Stakeholders

- Any developer
- @awilcoxa

### Specification



### Verification Steps

1. Open CHANGELOG.md
1. Verify release notes are accurrate

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
